src/components/bar.cpp

```diff
@@ -126,6 +130,8 @@ bar::bar(connection& conn, signal_emitter& emitter, const config& config, const
   m_opts.dimvalue = m_conf.get(bs, "dim-value", 1.0);
   m_opts.dimvalue = math_util::cap(m_opts.dimvalue, 0.0, 1.0);
 
+  m_opts.cursor_click = m_conf.get(bs, "cursor-click", ""s);
+  m_opts.cursor_scroll = m_conf.get(bs, "cursor-scroll", ""s);
```

Comment:
Also it would be useful to the user if polybar showed a warning, if the config uses an invalid cursor name.
We could maybe have a map that maps cursor names to name vectors. Like this we have all valid cursor names as well as their xcb names in one spot and `set_cursor` in `cursor.cpp` could just access the map instead of using a long if-elseif chain. This would also make it easier to, in the future, add new cursor names.